src/Payments.Mvc/Views/Invoices/Create.cshtml

```diff
@@ -15,6 +15,8 @@
     <script>
         window.team = @(Json.Serialize(ViewBag.Team));
 
+        window.accounts = @(Json.Serialize(ViewBag.Accounts));
```

Comment:
@jpknoll Should this be in the model?

Reply:
The list of available accounts isn't part of the invoice! Data for dropdowns we usually put in ViewBags.